testing/src/main/java/io/stargate/it/grpc/ExecuteQueryTest.java

```diff
@@ -184,6 +188,32 @@ public void simpleQueryWithPaging(@TestKeyspace CqlIdentifier keyspace)
     assertThat(rs.getPagingState()).isNotNull();
   }
 
+  @Test
+  @Disabled("Until issue #1577 fixed")
```

Comment:
Could separate into different PR (one with the fix), as an alternative. But thought this is easier way to share.